Lib/test/test_source_encoding.py

```diff
@@ -226,5 +226,18 @@ def check_script_output(self, src, expected):
         self.assertEqual(res.out.rstrip(), expected)
 
 
+class EncodingsTest(unittest.TestCase):
+
+    def test_bpo39337(self):
+        """
+        bpo-39337: similar to _Py_normalize_encoding(),
+        encodings.normalize_encoding() should ignore non-ASCII letters .
+        """
+        import encodings
+
+        out = encodings.normalize_encoding("utf\xE9\u20AC\U0010ffff-8")
```

Comment:
test_codecs would be a better place for tests on the encodings module.

Reply:
copy that, updated.